test/performance/latency-controller/main.go

```diff
@@ -105,7 +105,7 @@ func main() {
 	pflag.StringVar(&itsKubeconfigPath, "its-kubeconfig", "", "Path to ITS kubeconfig (empty = detect automatically)")
 	pflag.StringVar(&wecKubeconfigPaths, "wec-kubeconfigs", "", "Comma-separated paths to WEC kubeconfigs")
 	pflag.StringVar(&monitoredNamespace, "monitored-namespace", "default", "Namespace to monitor")
-	pflag.StringVar(&bindingPolicy, "binding-name", "nginx-singleton-bpolicy", "Binding policy name")
+	pflag.StringVar(&bindingName, "binding-name", "nginx-singleton-bpolicy", "Binding policy name")
```

Comment:
Here should be `bindingPolicy` name instead of `bindingName` - bindings is referring to another object in KubeStellar.